src/runtime/pkg/resourcecontrol/utils_linux.go

```diff
@@ -61,12 +68,44 @@ func ValidCgroupPathV2(path string, systemdCgroup bool) (string, error) {
 		return filepath.Clean(path), nil
 	}
 
+	if path == "" {
```

Comment:
Do you think that it is possible that because this method is not thread-safe, we are experiencing also other issues (like `timed out connecting to vsock`), as shown here: https://gist.github.com/norbjd/1773b2930227d89c347f223f1d9b4ff0#30-containers-in-parallel?

Reply:
Thanks for your suggestion, I will follow this to find the cause of the error and fix it.

Reply:
I still think random strings should be used, because the cgroup path is empty when nerdctl runs, it will cause conflicts between cgroups of multiple sandboxes, which itself is not correct.

Reply:
We discussed and get the results on Slack. The problem of `timed out connecting to vsock` was caused by running kata in a virtual machine. This problem was avoided on bare metal.